<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TutorialModule</class>
 <widget class="qMRMLWidget" name="TutorialModule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>338</height>
   </rect>
  </property>
  
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="volumePathTextBox"/>
   </item>
   
   <item>
    <widget class="QPushButton" name="loadVolumeButton">
     <property name="text">
      <string>Load Volume</string>
     </property>
    </widget>
   </item>

    <item>
     <layout class="QHBoxLayout" name="serverConfigQHBoxLayout">
     <item>
      <widget class="QLineEdit" name="shapePositionXTextBox">
       <property name="placeholderText">
        <string>Position X</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QLineEdit" name="shapePositionYTextBox">
       <property name="placeholderText">
        <string>Position Y</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QLineEdit" name="shapeSizeTextBox">
       <property name="placeholderText">
        <string>Size</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="drawCircleButton">
       <property name="text">
       <string>Circle</string>
      </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="drawSquareButton">
       <property name="text">
       <string>Square</string>
      </property>
      </widget>
     </item>
    </layout>
   </item>

   <item>
    <widget class="qMRMLSegmentEditorWidget" name="embeddedSegmentEditorWidget">
     <property name="autoShowSourceVolumeNode">
      <bool>true</bool>
     </property>
     <property name="maximumNumberOfUndoStates">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
